src/pandas_profiling/profile_report.py

```diff
@@ -124,6 +124,8 @@ def __init__(
 
         if config is not None:
             report_config = report_config.update(config.dict())
+            report_config.html.style._labels = config.html.style._labels
```

Comment:
Why would this not be updated by the above line? It sounds like this should be fixed in `report_config.update()`, not here.